macos/README.md

```diff
@@ -3,117 +3,10 @@ Wazuh macOS packages
-
-```shellsession
-$ sudo ./generate_wazuh_packages.sh -b v3.10.2 -j 4 -r 1 --notarize \
-    --keychain "/Users/your-user/Library/Keychains/login.keychain-db" \
-    --application-certificate "Your Developer ID Application" \
-    --installer-certificate "Your Developer ID Installer" \
-    --developer-id "your_apple_id@email.com" --keychain-password "login_password" \
-    --altool-password "temporary-password-for-altool"
-```
-
-The script will automatically sign the code and enable the _hardened runtime_, build the package and sign it, upload the package for its notarization and once it is notarized, the script will staple the notarization ticket to the package. Thanks to this, the package will be able to be installed in those hosts without an internet connection.
-
-Finally, you can find the notarization result in the `wazuh-packages/macos/request_result.txt`.
-
-### Additional information
-
-- [Enable hardened runtime (macOS)](https://help.apple.com/xcode/mac/current/#/devf87a2ac8f).
-- [About Code Signing](https://developer.apple.com/library/archive/documentation/Security/Conceptual/CodeSigningGuide/Introduction/Introduction.html).
-- [Code Signing Tasks](https://developer.apple.com/library/archive/documentation/Security/Conceptual/CodeSigningGuide/Procedures/Procedures.html#//apple_ref/doc/uid/TP40005929-CH4-SW26).
-- [Customizing the Notarization Workflow](https://developer.apple.com/documentation/security/notarizing_your_app_before_distribution/customizing_the_notarization_workflow?language=objc).
-- [Entitlements](https://developer.apple.com/documentation/bundleresources/entitlements).
-- [Hardened Runtime Entitlements](https://developer.apple.com/documentation/security/hardened_runtime_entitlements?language=objc).
-- [Resolving Common Notarization Issues](https://developer.apple.com/documentation/security/notarizing_your_app_before_distribution/resolving_common_notarization_issues).
-
-### Common issues
-
-- `xcrun: error: unable to find utility "altool", not a developer tool or in PATH`: this error appears when `xcrun` is unable to find altool. To solve it you will need to run:
-    ```
-    $ sudo xcode-select -r
-    ```
-    If this doesn't solve the issue, you will need to specify the path where Xcode is installed or unpacked:
-    ```
-    $ sudo xcode-select -s /path/to/Xcode.app
-    ```
-- `errSecInternalComponent` when running `codesign`: check the status of the login keychain. To solve it, you will need to close all the keychains and then run again the script.
-- `error: The specified item could not be found in the keychain`: this error may appear if `codesign` or `productsign` can't access to the Certificates, the private key or both. Check in the Keychain of your Mac hosts if they can be read by `codesign` and `productsign`.
+All info is in [Generate-osx-package.](https://documentation.wazuh.com/3.11/packaging/generate-osx-package.html)
```

Comment:
Please, replace "All info is in" by "Please, visit the following link for the full macOS packages building documentation:"

Reply:
Done in https://github.com/wazuh/wazuh-packages/pull/349/commits/90b419309b49288b9216ca08c55d61243f1e6bae.